test/testdata/lsp/field_edits.2.rbupdate

```diff
@@ -0,0 +1,8 @@
+# typed: true
```

Comment:
Do you want to assert that this takes the slow path? Because you can.

https://github.com/sorbet/sorbet#testing-incremental-typechecking